client/components/app.jsx

```diff
@@ -9,17 +9,19 @@ export default class App extends React.Component {
     };
   }
 
-  componentDidMount() {
-    fetch('/api/health-check')
-      .then(res => res.json())
-      .then(data => this.setState({ message: data.message || data.error }))
-      .catch(err => this.setState({ message: err.message }))
-      .finally(() => this.setState({ isLoading: false }));
-  }
+  // componentDidMount() {
+  //   fetch('/api/health-check')
+  //     .then(res => res.json())
+  //     .then(data => this.setState({ message: data.message || data.error }))
+  //     .catch(err => this.setState({ message: err.message }))
+  //     .finally(() => this.setState({ isLoading: false }));
+  // }
 
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <h1>{this.state.message.toUpperCase()}</h1>;
+    return (<LandingPage />);
   }
 }
+
+// return this.state.isLoading
+//   ? <h1>Testing connections...</h1>
+//   : <h1>{this.state.message.toUpperCase()}</h1>;
```

Comment:
remove commented out code since you can have it in the render after you've made the changes above.